src/v2/Apps/Consign/Components/GetPriceEstimate/ConsignPriceEstimateContext.tsx

```diff
@@ -151,20 +151,25 @@ function getActions(dispatch: Dispatch<Action>, relayEnvironment: Environment) {
         graphql`
           query ConsignPriceEstimateContext_ArtistInsights_Query(
             $artistInternalID: ID!
-            $medium: String!
           ) {
-            marketPriceInsights(artistId: $artistInternalID, medium: $medium) {
-              artistName
-              lowRangeCents
-              midRangeCents
-              highRangeCents
+            priceInsights(
+              artistId: $artistInternalID
+              sort: DEMAND_RANK_DESC
+              first: 1
+            ) {
+              edges {
+                node {
+                  artistName
```

Comment:
**question [non-blocking]:** display medium of price insight

I think you want to add `medium` here so that we can add this to the result card in the UI, no?

*format via [Conventional Comments](https://conventionalcomments.org/)*